drivers/serial/uart_npcx.c

```diff
@@ -365,6 +393,17 @@ static __unused void uart_npcx_rx_wk_isr(const struct device *dev,
 	npcx_uart_disable_access_interrupt();
 }
 
+#ifdef CONFIG_UART_CONSOLE_INPUT_EXPIRED
+static void uart_npcx_rx_refresh_timeout(struct k_work *work)
+{
+	struct uart_npcx_data *data = CONTAINER_OF(work, struct uart_npcx_data,
+						   rx_refresh_timeout_work);
+	const struct device *dev = data->uart_dev;
```

Comment:
See above, change `uart_npcx_pm_constraint_rel/set` to accept `struct uart_npcx_data` and you can eliminate this.

Reply:
Done.